test/unit/definitions/divorce/CaseTypeTab.test.js

```diff
@@ -1,13 +1,28 @@
 const expect = require('chai').expect;
 const assert = require('chai').assert;
 const { uniq, uniqWith, map, filter } = require('lodash');
+const { loadAllFilesIn } = require('../../utils/utils');
 
-const caseTypeTab = Object.assign(require('definitions/divorce/json/CaseTypeTab/CaseTypeTab'), {});
-const caseField = Object.assign(require('definitions/divorce/json/CaseField/CaseField'), {});
+let caseTypeTab = Object.assign(require('definitions/divorce/json/CaseTypeTab/CaseTypeTab'), {});
+let caseField = Object.assign(require('definitions/divorce/json/CaseField/CaseField'), {});
+
+let tabIds = uniq(map(caseTypeTab, 'TabID'));
+
+const nonProdCaseTypeTab = loadAllFilesIn('CaseTypeTab',
+    ['CaseTypeTab-prod', 'CaseTypeTab-deemed-and-dispensed-nonprod'])();
+
+const nonProdCaseField = loadAllFilesIn('CaseField',
+    ['CaseField-prod', 'CaseField-deemed-and-dispensed-nonprod'])();
```

Comment:
@yarpo How do you see this, it can also be assigned to a variable and called as a function but i am making it compact in this instance.